{"name": "test_simple_assume8", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\ntest_xdist.py:41: AssumptionFailure\n>>\tpytest.assume(1==0)\nAssertionError: assert False\n\ntest_xdist.py:43: AssumptionFailure\n>>\tpytest.assume(False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               test_xdist.py:41: AssumptionFailure\nE               >>\tpytest.assume(1==0)\nE               AssertionError: assert False\nE               \nE               test_xdist.py:43: AssumptionFailure\nE               >>\tpytest.assume(False)\nE               AssertionError: assert False\n\nD:\\Miniconda\\Lib\\site-packages\\six.py:718: FailedAssumption"}, "start": 1701430294610, "stop": 1701430294640, "uuid": "eda848a3-a351-45ce-bed2-83583a096823", "historyId": "28dc8653f7879e76ab1c036928faf351", "testCaseId": "28dc8653f7879e76ab1c036928faf351", "fullName": "test_xdist#test_simple_assume8", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}